media/js/base/mozilla-fxa-iframe.js

```diff
@@ -55,6 +62,15 @@ Mozilla.FxaIframe = (function() {
         // initialize GA event name
         _gaEventName = _config.gaEventName || 'fxa';
 
+        if (_config.distribution && _config.distribution !== 'default') {
+            _distHost = $('#fxa-iframe-config').data(_config.distribution.toLowerCase() + '-host');
```

Comment:
I think the case of the `distribution` value doesn't matter here, Firefox interprets ` data-MozillaOnline-host="... ` as ` data-mozillaonline-host="... ` and it always should be `.data(mozillaonlineHost)` with `Host` as suffix.

I'm making it more explicit by using ` data-{{ dist|lower }}-host="... ` in templates.